apps/web/src/api/api.client.ts

```diff
@@ -87,8 +87,14 @@ function buildUrl(url: string, absoluteUrl: boolean) {
 function getHeaders() {
   // TODO: change the way we get the clerk token
   const token = getToken();
-
-  return { Authorization: `Bearer ${token}` };
+  const lastEnvironmentId = getEnvironmentId();
+
+  return token
+    ? {
+        Authorization: `Bearer ${token}`,
+        'Novu-Environment-Id': lastEnvironmentId || '',
```

Comment:
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches and the blocking or non-blocking decorations to further communicate your intent.  -->
✏ **suggestion:** `getToken()` function returns either token or empty string - we could unify these two functions